Server/src/Main/ServerController.java

```diff
@@ -45,49 +45,53 @@ public void handleMessages() {
 				addBook(addBookRequest.getBook());
 				break;
 			case GET_ALL_BOOKS:
-				getAllBooksList();
+				sendAllBooksList();
 				break;
 			case GET_ALL_AUTHORS:
-				getAllAuthorsList();
+				sendAllAuthorsList();
+				break;
+			case GET_ALL_USERS:
+				sendAllUsersList();
 				break;
 			case SEARCH_BOOK:
-				GetBooksRequest getBookRequest = gson.fromJson(jsonMessage, GetBooksRequest.class);
-				searchBookInLibrary(getBookRequest.getTextForSearch());
+				SearchBookRequest searchBookRequest = gson.fromJson(jsonMessage, SearchBookRequest.class);
+				searchBookInLibrary(searchBookRequest.getTextForSearch());
 				break;
 			case USER_CHECK:
 				UserCheckRequest userCheckRequest = gson.fromJson(jsonMessage, UserCheckRequest.class);
 				checkUser(userCheckRequest.getLogin(), userCheckRequest.getPassword());
 				break;
 			case LOGIN_CHECK:
-				LoginCheckRequest loginCheckRequest = gson.fromJson(jsonMessage, LoginCheckRequest.class);
-				checkLogin(loginCheckRequest.getNewLogin());
+				IsLoginExistRequest isLoginExistRequest = gson.fromJson(jsonMessage, IsLoginExistRequest.class);
+				checkLogin(isLoginExistRequest.getNewLogin());
 				break;
 			case ADD_USER:
-				addUser();
+				AddUserRequest addUserRequest = gson.fromJson(jsonMessage, AddUserRequest.class);
```

Comment:
change registration logic: add fields "name" "surname" to "users" table and AddUserRequest